gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -6635,6 +6646,11 @@ subroutine interstitial_phys_reset (Interstitial, Model)
     Interstitial%dudt_mtb        = clear_val
     Interstitial%dudt_ogw        = clear_val
     Interstitial%dudt_tms        = clear_val
+!-- GSD drag suite
```

Comment:
If these are only used for the GSD drag suite, then please initialize them only if the drag suite is used.